src/docs/Components/OptionGroup/OptionCard/code.mdx

```diff
@@ -0,0 +1,83 @@
+---
+title: 'Code'
+order: 2
+---
+
+## Import
+
+Import the settings and the option-card `scss` files for the option card component.
+
+```scss
+@import 'settings-tools/_all-settings';
+@import '_c.option-card'; // mandatory
+```
+
+## Usage
+
+The creation of an option card component in your code is done using the following HTML structure:
+
+```HTML
+  <div class="mc-option-card">
+    <input 
+      type="radio" 
+      name="option-group" 
+      id="radio-01" 
+      class="mc-radio__input mc-option-card__input" 
+      />
+    <label for="radio-01" class="mc-option-card__label"></label>
+    <div class="mc-option-card__content">
+      <!-- Here your content -->
+    </div>
+  </div>
+```
+
+## Implement as a group
+
+In order to group and organize a list of `option-card` you must wrap them in a HTML element using `mc-option-group` class.
+
+This class has to be define in the user environment. In fact we let the end-user choose the layout that suit their needs. 
+We highly recommand to use it with `flexbox`. 
+
+```scss
+  .mc-option-group {
+    display: flex;
+    justify-content: space-between | space-evenly;
+    ...
+  }
+```
+
+<Highlight theme="warning" title="Be aware">
+
+Please, make sure your option-card are wrapped around a flex div. 
+
+</Highlight>
+
```

Comment:
An explanation is missing about the fact that it is up to the user to define the dimensions of the element `mc-option-card__content`